vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/BiodataVariantToVariantContextConverterTest.java

```diff
@@ -400,6 +402,43 @@ public void twoStudiesConflictingNamesSingleVariant() {
         checkVariantContext(variantContext, CHR_1, 1000, 1000, "T", "G", variant.getSourceEntries(), true);
     }
 
+
+    @Test
+    public void csqAnnotation() {
+        // create variant
+        VariantSource variantSource = createTestVariantSource(STUDY_1);
+        String variantLine = String
+                .join("\t", CHR_1, "1000", "id", "C", "A", "100", "PASS", ".", "GT", "0|0", "0|0", "0|1", "1|1", "1|1",
+                      "0|1");
+        List<Variant> variants = variantFactory.create(variantSource, variantLine);
+        assertEquals(1, variants.size());
+        // populate variant with test csq data
```

Comment:
Can you separate the blocks in this method with blank lines, to improve readability? (crete variant, add csq data, export and test)